crates/bevy_math/src/direction.rs

```diff
@@ -243,6 +243,15 @@ impl Dir2 {
     pub fn rotation_to_y(self) -> Rot2 {
         self.rotation_from_y().inverse()
     }
+
+    /// Returns `self` after an approximate normalization and , assuming the value is already nearly normalized.
```

Comment:
Seems like an unfinished thought here and all the other methods
`...normalization and , assuming...`

Reply:
...that's what happens when you rewrite a sentence too many times.

should've been 
```
/// Returns `self` after an approximate normalization, assuming the value is already nearly normalized.
```